### DISCUSSION TITLE
Whats the limit to how many devices can be plugged in

### DISCUSSION THREAD

**Discussion by Mark62222** [dorssel/usbipd-win#932] [Q&A]
and do you know if theres a limit to the usbip on linux to how many devices it can attach

**Comment by dorssel** [comment]
See https://github.com/torvalds/linux/blob/e67572cd2204894179d89bd7b984072f19313b03/drivers/usb/usbip/vhci.h#L75-L89

TL;DR: default kernel has 1 controller, with 8 normal/high speed USB2 ports, and 8 superspeed USB3 ports.

You need to recompile the kernel to support more.
